tests/graph/test_skeleton_graph.py

```diff
@@ -31,3 +31,11 @@ def test_skeleton_graph_json_round_trip(simple_t_skeleton_graph, tmp_path):
 
     # check that the graphs are equal
     assert simple_t_skeleton_graph == new_skeleton_graph
+
+def test_skeleton_graph_to_directed(simple_t_skeleton_graph):
+    """Test converting a SkeletonGraph to a directed graph."""
+    directed_graph = simple_t_skeleton_graph.graph.to_directed()
```

Comment:
Does this work? It looks like `origin` is a required argument

Reply:
Ah I see. I think it was running because the `origin` parameter was assigned `int` instead of typed as `int` (i.e., `=` instead of `:`).

Reply:
woops, that was because I called the networkx function on the graph and not on the skeleton object. Good catch